docs/source/release_notes.rst

```diff
@@ -22,6 +22,7 @@ Future Release
         * Refactor ``EntitySet.query_by_values`` to work with Woodwork dataframes (:pr:`1467`)
         * Replace ``list_variable_types`` with ``list_logical_types`` (:pr:`1477`)
         * Allow deep EntitySet equality check (:pr:`1480`)
+        * Add function to list semantic tags (:pr:`1486`)
```

Comment:
We should add a section to `What's New in this Release` that describes the new list logical types and list semantic tags

Reply:
Added section